examples/graph_sage_unsup_ppi.py

```diff
@@ -20,7 +20,7 @@
 # Group all training graphs into a single graph to perform sampling:
 train_data = Batch.from_data_list(train_dataset)
 loader = LinkNeighborLoader(train_data, batch_size=2048, shuffle=True,
-                            neg_sampling_ratio=0.5, num_neighbors=[10, 10],
+                            neg_sampling_ratio=1.0, num_neighbors=[10, 10],
```

Comment:
Update example to use `neg_sampling` and point to this example in `link_loader` docs.

Reply:
Will update examples once PyG 2.2 lands to avoid breaking examples for all users with PyG 2.1